lib/heex_formatter/phases/format.ex

```diff
@@ -109,6 +129,10 @@ defmodule HeexFormatter.Phases.Format do
   end
 
   # eex_tag_render represents <%=
+  defp token_to_string({:eex_tag_render, tag, _meta}, %{mode: :script} = state) do
+    %{state | html: state.html <> String.trim(tag)}
+  end
+
```

Comment:
```suggestion
  defp token_to_string({:eex_tag_render, tag, _meta}, %{mode: :script} = state) do
    %{state | html: state.html <> String.trim(tag)}
  end

```

We don't need this either if we are not indenting JS tag.

Reply:
We need it, else it will put spaces around.

Reply:
I wouldn't worry about it since we decided to not format JS code and it is inside of it 🙌🏻